src/main/java/com/spring/rest/model/CarCharacteristics.java

```diff
@@ -1,17 +1,15 @@
 package com.spring.rest.model;
 
-import java.util.Calendar;
-import java.util.Date;
-
 /**
  * The car characteristics model.
  * This model includes year of car manufacture, car mileage, fuel type of car, transmission type and car model.
  * Each car includes this model.
+ *
  * @author Katuranau Maksimilyan
  */
 public class CarCharacteristics {
     private int carId;
-    private Date year;
+    private int year;
```

Comment:
What if car has also month?

Reply:
it is never written

Reply:
I propose to have month and year. DB has DATE structure w/o time. Look at it